mapie/estimators.py

```diff
@@ -358,10 +403,9 @@ def _fit_and_predict_oof_model(
             - [3]: Validation data indices, of shapes (n_samples_val,)
         """
         X_train, y_train, X_val = X[train_index], y[train_index], X[val_index]
-        if sample_weight is not None and supports_sw:
-            estimator.fit(X_train, y_train, sample_weight=sample_weight[train_index])
-        else:
-            estimator.fit(X_train, y_train)
+        if sample_weight is not None:
+            sample_weight = sample_weight[train_index]
```

Comment:
Attention, il se peut que cette ligne de code modifie `sample_weight` in-place. Je dirais plutôt : 
```
if sample_weight is not None:
    sample_weight_train = sample_weight[train_index]
estimator = self._fit_estimator(estimator, X_train, y_train, supports_sw, sample_weight_train)

Reply:
Corrigé. J'ai du coup défini juste avant `sample_weight_train = same_weight` .